src/log-io.js

```diff
@@ -53,19 +75,34 @@ class LogIO {
     }
   }
 
-  static async fromEntryHash (ipfs, entryHash, id, length = -1, exclude, onProgressCallback) {
```

Comment:
Should we keep this but deprecate it? I'm not sure if it's part of the public API.

Reply:
I believe we can deprecate this. @shamb0t can you confirm?

Reply:
@haadcode yes I must restore the function and declare it as deprecated.